packages/longhorn/longhorn-1.4/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,19 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,3 +1,16 @@
++annotations:
++  catalog.cattle.io/auto-install: longhorn-crd=match
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/display-name: Longhorn
++  catalog.cattle.io/kube-version: '>= 1.21.0-0'
```

Comment:
Ideally, this should be a closed ended range that excludes k8s 1.25 (`'>= 1.21.0-0 < 1.25.0-0'`). The reason being that regardless of longhorn supporting k8s 1.25, or longhorn having the PSPs disabled for 1.25, it will still be installable and there are upgrade/installation use cases around PSPs/PSAs to consider.

More info on how we are handling PSPs for 1.25: https://github.com/rancher/rancher/issues/40331

cc @prachidamle @MKlimuszka

Reply:
@PennyScissors 

> Ideally, this should be a closed ended range that excludes k8s 1.25 ('>= 1.21.0-0 < 1.25.0-0')

Sorry, I don't get this point. This chart will run on kubernetes version >= 1.21 including 1.25. Why should we limit it this way?

Regarding to the PSP/PSA stuff. This Longhorn chart already handles it. The Longhorn team doesn't need to use the new Rancher way of dealing with PSP which is creating the new setting `.Values.global.cattle.psp.enabled`.


@longhorn-qa (@chriscchien) Could you please test 3 scenarios?:
1. Install Longhorn 1.4.0 in Kubernetes 1.24 cluster 
2. Install Longhorn 1.3.0 in Kubernetes 1.24 cluster. Upgrade Longhorn to this 1.4.0 version. Upgrade Kubernetes cluster to 1.25.
3. Install Longhorn 1.4.0 in a fresh Kubernetes 1.25 cluster

Reply:
I wasn’t aware the longhorn chart already handles the PSPs use cases for 1.25. lgtm